subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_pgps

```diff
@@ -93,4 +93,26 @@ config NRF_CLOUD_PGPS_SOCKET_RETRIES
 	help
 	  This sets the maximum number of times to retry a download.
 
+config NRF_CLOUD_PGPS_PARTITION
```

Comment:
If a user enables this, and there is no other image in the build (such as SPM), then partition manager will not be called, as it is only invoked by default when multiple images are included in the build. That would (IIRC) cause the build to fail.

An solution to this would be to have this config depend on `PARTITION_MANAGER_ENABLED`. This is a read-only config which is enabled whenever partition manager is enabled.

To enable partition manager in a single image build, you can set the `CONFIG_PM_SINGLE_IMAGE` option.

Reply:
Thanks for that.  I will try to test this and confirm it is an issue and apply your fix.

Reply:
I tried disabling `CONFIG_SPM`, and besides compiler errors on samples/gnss/src/assistance.c and subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c, three modem lib files also fail to compile:

1. sdk-nrf/lib/nrf_modem_lib/shmem_sanity.c
2. nrf_modem_lib.c
3. nrf_modem_os.c
 
because `pm_config.h` does not exist.

I added this to `Kconfig.nrf_cloud_pgps` and it now works correctly.

	select PM_SINGLE_IMAGE if !SPM